v2/types.go

```diff
@@ -19,10 +19,36 @@ import (
 	"encoding/json"
 	"fmt"
 	"net"
+	"net/url"
 	"strings"
 	"time"
 )
 
+const MaxInfoLength = 255
```

Comment:
ha! this is a Pascal string? 255 sounds reasonable.

Reply:
I just wanted to keep the description at a reasonable length.  We can increase that any time.